### ISSUE TITLE
Tiled Not Opening - "Tiled Not Responding"

### ISSUE THREAD

**Issue by TsaEIT** [bjorn/tiled#2091]
Sorry to bother you,

But, recently whenever I attempt to open a .tmx or even open Tiled normally (from the Windows Menu) the window shows up white, waits for 3-5 seconds and closes (don't use that exactly). I attempted reinstalling Tiled, with no luck, I've rebooted my computer, and I can't think of anything else.

Some specs that I could give you would be:
OS: Windows 10 Pro
  Version: 1803
Processor:
  Manufacturer: Intel
  Core: i5-3450
Installed RAM: 8 GB (7.89 usable)
64-bit

**Comment by bjorn** [comment]
Recently there have been a few reports like this, and it appears to be related to the Python plugin, which crashes on load when certain versions of Python are installed. Can you please tell me:

* The exact version of Tiled you installed (including architecture)
* Which Python versions you have installed (including architecture)

With this information I may be able to reproduce the problem to see if I can find a solution. Please also try to delete the `python.dll` that is part of your Tiled install, and tell me whether it fixes the problem.

If I can't find a solution I will at least disable the Python plugin by default.

**Comment by TsaEIT** [comment]
My Current Tiled Version Is 1.2.2
I currently have Python 2.7.11 and 3.6.5
  but due to Tiled working earlier, I assume it was running 3.6.5.

**Comment by TsaEIT** [comment]
Python 2 being 32-bit
Python 3 being 64-bit

**Comment by TsaEIT** [comment]
Once again sorry for the string of comments, but deleting `python.dll` seems to have worked.
Thanks!

**Comment by bjorn** [comment]
> Python 2 being 32-bit
> Python 3 being 64-bit

And Tiled being? :-)

**Comment by TsaEIT** [comment]
1.2.2 I believe.

**Comment by TsaEIT** [comment]
I tried removing the python.dll file, and it seems to work perfectly, thanks for the help.

**Comment by bjorn** [comment]
> 1.2.2 I believe.

I was wondering whether it was the 32-bit or 64-bit version.

**Comment by TsaEIT** [comment]
64-bit

**Comment by ASB2** [comment]
I had this same issue, it was resolved by deleting python.dll. Im using tiled Version 1.2.3 64bit with python version 3.7.3

**Comment by bjorn** [comment]
I've pushed a change that will disable the Python plugin by default in Tiled 1.2.4 (not sure when I'll release this yet). Though ideally we'd find out how to avoid this crash from happening in the first place...

**Comment by albin-johansson** [comment]
Hi! 

I recently ran into this annoying issue, so I thought that I'd share what I did to solve it locally. The problem first arose when I upgraded my Tiled version (I believe from 1.5.0 to 1.7.0, but the issue remained when I tried 1.6.0 as well). 

So I tried the aforementioned workaround of removing the `python.dll` file from `.../Tiled/plugins/tiled`, but that didn't help. So I eventually got around to removing the entire Tiled entry from the registry, which seems to have solved issue. Since my issues arose when I upgraded the Tiled version, the problem might have something to do with registry entries getting corrupted somehow? Even if it admittedly does sound like a strange cause of "Not Responding..." errors, but I don't know enough about how Qt handles registry data through `QSettings`.

**tl;dr** I did *not* have to remove `python.dll` file to solve the issue, I simply removed the Tiled entry from the registry and booted the application.

* OS: 64-bit Windows 10 Home (Version 20H2 Build 19042.1083)
* Tiled 1.7.0 (64-bit)
* Python: 3.9.6 (64-bit)

**Comment by bjorn** [comment]
@albin-johansson Hmm, that's interesting, but it's the first time I hear about it. Just in case, could it be that you had enabled hardware accelerated rendering (OpenGL)? Because this can also cause hangs on startup I think, and the setting is stored in the registry.

If you try it out and it breaks Tiled, note that you can also start Tiled with the command-line parameter `--disable-opengl` to disable it again.

**Comment by albin-johansson** [comment]
Yeah, I have usually had hardware accelerated rendering enabled, but I don't remember for sure. If I run into the issue again, I'll be sure to try to to disable it.
